dbt_meshify/storage/dbt_project_editors.py

```diff
@@ -279,7 +277,8 @@ def move_resource_yml_entry(self, resource: Resource) -> ChangeSet:
         resource_entry = self.load_resource_yml(
             current_yml_path, resource.name, resource.resource_type, source_name
         )
-
+        if not resource_entry:
```

Comment:
you know i wrote this thinking about the earlier feedback about returning early but i almost feel like only extending the change set if there is a resource entry might actually read nicer here

Reply:
>i almost feel like only extending the change set if there is a resource entry might actually read nicer here

I'm inclined to agree 👍🏻

Reply:
updated!